src/lib/assets/index.ts

```diff
@@ -28,6 +29,68 @@ import { watcher, moduleSecret } from "./pods";
 
 import { clusterRoleBinding, serviceAccount, storeRole, storeRoleBinding } from "./rbac";
 import { createDirectoryIfNotExists } from "../filesystemService";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function toYaml(obj: any): string {
+  return dumpYaml(obj, { noRefs: true });
+}
+
+function webhookYaml(
```

Comment:
```suggestion
function createWebhookYaml(
```

This function reads less like a property name if we change the function name, which aids clarity in my opinion